addons/uom/models/uom_uom.py

```diff
@@ -199,3 +232,29 @@ def _filter_protected_uoms(self):
             return self.browse()
         else:
             return self.browse(set(linked_model_data.mapped('res_id')))
+
+    @api.depends('uom_type', 'factor')
+    def _compute_ratio(self):
```

Comment:
the order of method to respect in the guideline (https://www.odoo.com/documentation/14.0/reference/guidelines.html#symbols-and-conventions)